### PR TITLE
[Y2K-361] Fix verification modal so there's a back button

### PR DESCRIPTION
**Pull request by jzila**
cc @keybase/y2ksquad

### CODE REVIEW

File: shared/settings/account/add-modals.tsx [hunk 1]
```diff
@@ -222,9 +222,12 @@ export const VerifyPhone = () => {
...
       header={{
         hideBorder: true,
+        leftButton: Styles.isMobile ? (
+          <Kb.BackButton onClick={onClose} iconColor={Styles.globalColors.white} />
+        ) : null,
         style: styles.blueBackground,
         title: (
-          <Kb.Text type="BodySmall" negative={true}>
+          <Kb.Text type="BodySmall" negative={true} center={true}>
             {displayPhone || 'Unknown number'}
```
Comment (jakob223) [inline shared/settings/account/add-modals.tsx]: 🐑 🐑 🐑 do you want to recapitalize the N?

Comment (jzila) [reply shared/settings/account/add-modals.tsx]: Naw, I think our standard is not to capitalize.

Comment (jakob223) [reply shared/settings/account/add-modals.tsx]: o I had thought your previous iteration of this PR capitalized it.

Comment (jakob223) [reply shared/settings/account/add-modals.tsx]: oh I see I looked at initially before you had undone that

**Pull request merged**